src/features/dashboard/ui/PariticipantsList.tsx

```
@@ -17,8 +19,43 @@ const ParticipantsList = ({ listType, selectedFilter = [], participants }: Parti
     initializeParticipants,
     toggleAll,
     toggleParticipant,
+    selectedTicketId,
+    selectedOrderId,
+    setSelectedTicketId,
+    setSelectedOrderId
   } = useParticipantStore();
 
+  const { data } = usePersonalTicketOptionAnswers(selectedTicketId);
+  const [isModalOpen, setModalOpen] = useState(false);
+  const [selectedOrder, setSelectedOrder] = useState<any>(null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)